compiler/testData/diagnostics/testsWithStdLib/inference/intersectionInputType.kt

```diff
@@ -1,3 +1,4 @@
+// FIR_IDENTICAL
```

Comment:
This isn't the result of your change?

Reply:
This is not. Let me make it a separate commit.